AEM/optimum-habana-perf.md

```diff
@@ -1,48 +1,57 @@
-```bash
-git clone -b v1.16.0 https://github.com/huggingface/optimum-habana.git
-cd optimum-habana/examples/text-generation
-```
+## Build and Run the Benchmark Docker instance 
+
+### Get Dockerfile and Benchmark scripts
+First, get the Dockerfile and Benchmark scripts from Gaudi-Tutorial GitHub Repositroy following below command.  
 
-### Docker Run
-After building the Docker image, run the following command to start a Docker instance, which will open in the text-generation folder inside the docker instance.
 ```bash
-docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none   --cap-add=ALL --privileged=true  --net=host --ipc=host  -v "$PWD/../../":/workspace --workdir  /workspace/examples/text-generation  vault.habana.ai/gaudi-docker/1.20.0/ubuntu24.04/habanalabs/pytorch-installer-2.6.0:latest
+git clone https://github.com/HabanaAI/Gaudi-tutorials.git
+cd Gaudi-tutorials/PyTorch/Hugging_Face_pipelines/Benchmarking_on_Optimum-habana_with_fp8
 ```
-{:.note}**NOTE:** The Huggingface model file size might be large, so it is recommended to use an external disk as the Huggingface hub folder. Export the HF\_HOME environment variable to the external disk and then export the mount point into the Docker instance. ex: "-e HF\_HOME=/mnt/huggingface -v /mnt:/mnt"
-
-### Install required packages inside docker
-First, install the optimum-habana:
+### Docker Build
+To build the image from the Dockerfile, please follow below command to build the optimum-habana-text-gen image.
 ```bash
-pip install --upgrade-strategy eager optimum[habana]
+docker build --no-cache -t optimum-habana-text-gen:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
 ```
-
-Second, install the requirements:
+### Docker Run
+After docker build, users could follow below command to run and docker instance and users will be in the docker instance under text-generation folder.
 ```bash
-pip install -r requirements.txt
+docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none   --cap-add=ALL --privileged=true  --net=host --ipc=host optimum-habana-text-gen:latest
 ```
+> [!NOTE]
+> The Huggingface model file size might be large, so we recommend to use an external disk as Huggingface hub folder. \
+> Please export HF_HOME environment variable to your external disk and then export the mount point into docker instance. \
+> ex: "-e HF_HOME=/mnt/huggingface -v /mnt:/mnt"
+
+# Run Benchmark with Benchmark.py
+Benchmark script will run all the models with different input len, output len and batch size and generate a report to compare all published numbers in [Gaudi Model Performance](https://www.intel.com/content/www/us/en/developer/platform/gaudi/model-performance.html).  
 
-For `run_lm_eval.py`:
+## Gaudi3
+Different json file are provided for different Gaudi Software version like 1.19 and 1.20 on Gaudi3.
```

Comment:
Change "file" to "files"; change "version" to "versions". You may want to change "Gaudi Software" to "Intel Gaudi software", and place the correct copyright symbols on the Intel and the Gaudi.